web/src/translation/translation.ts

```diff
@@ -32,29 +32,18 @@ export const translateIfExists = (key: string) => {
   return i18next.exists(key) ? i18next.t(key) : '';
 };
 
-export const getZoneName = (zoneCode: string) =>
-  translateIfExists(`zoneShortName.${zoneCode}.zoneName`);
+export const getFullZoneName = (zoneCode: string) => {
+  return translateIfExists(`zoneShortName.${zoneCode}.zoneName`);
+};
+export const getZoneName = (zoneCode: string) => {
+  const displayName = translateIfExists(`zoneShortName.${zoneCode}.displayName`);
+  const fullName = getFullZoneName(zoneCode);
+  return displayName || fullName;
+};
```

Comment:
If you want to do a micro optimisation size wise you can extract the `zoneShortName.${zoneCode}.zoneName` to arrow function to avoid the extra string size from declaring it twice 😅